arwen/contexts/runtime.go

```diff
@@ -163,8 +164,7 @@ func (context *runtimeContext) makeInstanceFromCompiledCode(codeHash []byte, gas
 
 	context.instance = newInstance
 
-	idContext := arwen.AddHostContext(context.host)
-	context.instance.SetContextData(idContext)
+	context.instance.SetContextData(uintptr(unsafe.Pointer(&context.host)))
```

Comment:
Extract pointer of `context.host` as a variable, for readability:
```suggestion
        hostReference := uintptr(unsafe.Pointer(&context.host))
	context.instance.SetContextData(hostReference)
```